data/json/items/containers/generic.json

```diff
@@ -43,5 +43,34 @@
     "use_action": { "type": "deploy_furn", "furn_type": "f_trashcan" },
     "symbol": ")",
     "color": "light_gray"
+  },
+  {
+    "id": "trashcan_large",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "industrial trash can" },
+    "looks_like": "f_trashcan",
+    "description": "A really big trashcan in which you can store, you guessed it, trash, ready for the garbage truck to pick up.  Made out of thick plastic, this can was meant for household or industrial use, something you roll up to the curb once a week.",
+    "weight": "17700 g",
+    "//1": "Weight taken from https://www.uline.com/Product/Detail/H-7938BL/Trash-Cans-with-Wheels/Uline-Trash-Can-with-Wheels-95-Gallon-Black?pricode=WB3072&utm_source=Bing&utm_medium=pla&utm_term=H-7938BL&utm_campaign=Facilities%2BMaintenance&utm_source=Bing&utm_medium=pla&utm_term=H-7938BL&utm_campaign=Facilities%2BMaintenance&msclkid=d3c8827d02691eb87066e72c65ff5790",
```

Comment:
Link says 33lb which is 15kg rounding up like 50g. I can quibble about whether all houses would have the largest size but this is better than nothing.